.github/workflows/continuous-integration.yml

```diff
@@ -142,6 +142,24 @@ jobs:
     - name: Build the library
       run: make
 
+  buildsamples:
+    name: Build Samples
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+    - uses: actions/checkout@v3
+      with:
+        submodules: false
+    - name: Initialize the directory
+      uses: ./.github/actions/setup-libcgroup
+    - name: Reconfigure libcgroup with samples enabled
+      run: CFLAGS="$CFLAGS -Werror" ./configure --sysconfdir=/etc --localstatedir=/var --enable-code-coverage --enable-opaque-hierarchy="name=systemd" --enable-python --enable-samples
+    - name: Build the library
+      run: make
+
```

Comment:
Should we have a step `buildtest` combining both `buildsamples` and `withoutsystemd`, to avoid spawning a new VM for build tests?

Reply:
That's a good plan :+1: